core/crypto/src/util.rs

```diff
@@ -92,6 +94,25 @@ impl<
     }
 }
 
+impl PublicKey {
+    /// Create the implicit public key from an implicit account ID.
+    ///
+    /// Panics if the given account id is not a valid implicit account ID.
+    /// See [`near_account_id::AccountId#is_implicit`] for the definition.
```

Comment:
This was fine for a test util, but should be changed to return a `Result` for code that's made available for production code.

Reply:
Updated to return `ImplicitPublicKeyError::AccountIsNotImplicit`.